releases/point-2022-3.md

```diff
@@ -0,0 +1,81 @@
+---
+description: This article lists improvements and bug fixes in SysKit Point version 2022.3.
+--- 
+
+# SysKit Point 2022.3
+
+[Schedule a demo](https://www.syskit.com/products/point/request-a-demo/) and [tell us what you think.](https://www.syskit.com/company/contact-us/)
+
+**Product version:** 2022.3.0
+
+**Build number:** 120
+
+**Release date:** May 13, 2022
+
+## Get the Latest Version
+
+* **New to SysKit Point?** Get the latest SysKit Point version from [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/syskitltd.syskit_point).<br/>
+    Find all steps described in detail in the [deployment article](../installation/deploy-syskit-point.md).
+    
+* **Already using SysKit Point?** Upgrade to the latest SysKit Point version. <br/>
+[![](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fsyskitassetsstorage.blob.core.windows.net%2Fpoint%2FUpdateFilesARM%2FPointUpdateTemplate.json)
+
+See detailed step-by-step instructions on how to upgrade SysKit Point in the [upgrade article](../installation/upgrade-syskit-point.md).
+
+## Features
+* **SysKit Point provisioning templates allow choosing metadata columns** to include additional attributes when generating reports.
+    * The metadata columns available in the dropdown menu provide:
+        * **Setting custom values**
+        * **Users can define and fill in the value when creating a request**
+        * **The ability to create and edit new metadata columns**
+    * **Admins can define values** for metadata columns for every site, team or group on the Overview screen.
+        *The pre-defined metadata **set by the admin will be shown to the user as read-only**.
+    * For more information on **SysKit Point metadata, check out this article**.
+* **Service accounts can be added to an ignore list**.
+    * Setting up an ignored service account means:
+        * The Service Account page hits and views **will be ignored when detecting any workspace activity**.
+        * The Service Account audit logs **won’t be stored**.
+        * The Service Account changes **will be ignored and won’t trigger alerts**.
+    * Read more about **how to Ignore Service accounts here**.
+* **Added the ability to remove Sharing links** when generating the report.
+    * Select All feature available for removing all links attached to a certain Site.
```

Comment:
CIM (bivši Trace) će od ove verzije biti integriran u Point tako da je promjena dosta veća od dodavanja CIM settingsa budući da su mergeana 2 proizvoda zapravo. 
ovo ćemo Hrvoje i ja riješiti pa za sad ostavi kako je, al čisto da imaš više info o tome što se tu zapravo radilo :)